satellite/console/consoleweb/server.go

```diff
@@ -530,10 +535,40 @@ func (server *Server) initializeTemplates() (err error) {
 }
 
 // stripPort strips the port from hostport string if any.
-func stripPort(hostport string) string {
-	i := strings.Index(hostport, ":")
+func stripPort(hostport string) (string, error) {
+	i := strings.LastIndex(hostport, ":")
```

Comment:
What if it's ipv6 without port?

Reply:
is it even possible?

Reply:
It's possible for ipv4 addresses so I think it should be possible for ipv6.

Reply:
also `ipv6` itself contains `:` as a delimiter

Reply:
https://en.wikipedia.org/wiki/IPv6_address#Literal_IPv6_addresses_in_network_resource_identifiers

Reply:
Well I am not an expert here if you know some corner cases which won't work here, I will be happy to rewrite the function so it can work as it should

Reply:
`http://[2001:db8:85a3:8d3:1319:8a2e:370:7348]/` contains `:`, so we go to `ipv6` handling part, right?

Reply:
By the way, I have read that by default browser doesn't share the same cookies between subdomains, without explicitly asking it to do so. So all this changes, adding host to cookie key seems redundant